aspnet/fundamentals/configuration.rst

```diff
@@ -4,7 +4,7 @@ Configuration
 =============
 `Steve Smith`_, `Daniel Roth`_
 
-ASP.NET Core supports a variety of different configuration options. Application configuration data can come from files using built-in support for JSON, XML, and INI formats, as well as from environment variables. You can also write your own :ref:`custom configuration provider <custom-config-sources>`.
+ASP.NET Core supports a variety of different configuration options. Application configuration data can come from files using built-in support for JSON, XML, and INI formats, as well as from environment variables. You can also write your own :ref:`custom configuration source <custom-config-sources>`.
```

Comment:
If this is supposed to be nearly complete list you may want to mention command line arguments and in memory provider.